proto/cosmos/base/v1beta1/coin.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
```

Comment:
for these third party protos, have a third_party dir, as in: https://github.com/cosmos/cosmos-sdk/tree/master/third_party ?

Reply:
Done